webapp/components/wpt-runs.js

```diff
@@ -374,6 +433,42 @@ class WPTRuns extends WPTFlags(SelfNavigation(LoadingState(TestRunsUIBase))) {
       }
     }
   }
+
+  _showRuns(diff) {
+    const url = new URL('/results', window.location);
+    for (const id of this.selectedRuns) {
+      url.searchParams.append('run_id', id);
+    }
+    if (diff) {
+      url.searchParams.set('diff', true);
+    }
+    window.location = url;
+  }
+
+  handleSelectRun(e) {
+    const id = e.target.getAttribute('data-run-id');
+    if (this.selectedRuns.find(r => r === id)) {
```

Comment:
Can this be just `e.target.hasAttribute('selected')`? Is it possible for that to be out of sync with the condition here?

Reply:
It's safer to use the set as the source of truth, but shouldn't be possible to be out of sync.